src/routes/routes.tsx

```diff
@@ -65,81 +69,77 @@ const routesObject: Record<
     authenticatedRoute: true,
     title: 'Send',
     component: () => (
-      <PrivateKeyCheckWrapper>
+      <FeaturePageLayout title='Send'>
         <Send />
-      </PrivateKeyCheckWrapper>
+      </FeaturePageLayout>
     )
   },
   [RouteNamesEnum.sign]: {
     path: RouteNamesEnum.sign,
     title: 'Sign',
-    component: () => (
-      <PrivateKeyCheckWrapper>
-        <Sign />
-      </PrivateKeyCheckWrapper>
-    )
+    component: () => <Sign />
   },
   [RouteNamesEnum.signMessage]: {
     path: RouteNamesEnum.signMessage,
     title: 'Sign Message',
     component: () => (
-      <PrivateKeyCheckWrapper>
+      <FeaturePageLayout title='Sign Message'>
         <SignMessage />
-      </PrivateKeyCheckWrapper>
+      </FeaturePageLayout>
     )
   },
   [RouteNamesEnum.sovereignTransfer]: {
     path: RouteNamesEnum.sovereignTransfer,
     title: 'Sovereign Transfer',
     component: () => (
-      <PrivateKeyCheckWrapper>
+      <FeaturePageLayout title='Sovereign Transfer'>
         <SovereignTransfer />
-      </PrivateKeyCheckWrapper>
+      </FeaturePageLayout>
     )
   },
   [RouteNamesEnum.issueToken]: {
     path: RouteNamesEnum.issueToken,
     title: 'Issue Token',
     component: () => (
-      <PrivateKeyCheckWrapper>
+      <FeaturePageLayout title='Issue Token'>
```

Comment:
titles are dublicated
I think it was better with a new prop called `layout: LayoutsEnum.FeaturePage`
then a switch that wrapps the component with the correct layout and inserts the title as prop